### PR TITLE
feat(balance): Adding a fuel pod to afterburner Raven

### PR DESCRIPTION
**Pull request by LorenzoBolla**
## Summary
The Raven has an afterburner variant which does not have a ramscoop equipped, but has the leftover space for one! Since equipping a ramscoop means it can get fuel it used back, and it has space leftover for it, there's no reason for it not to have one. 

Probably shouldn't have chosen to be succinct over delving deep into my reasoning for this change. I have explained in detail my reasoning in the comment below, replying to Zitchas. Thank you, Z, for inducing that!

Due to the discussed points listed below, the ramscoop has been replaced for a fuel pod.

### CODE REVIEW

**Comment by Zitchas** [comment]
While I don't object to this, there's more to determining a loadout than just "can it fit" or "would it be optimal to have one".

A few additional questions to ponder before we decide to make this change:
1. Can the NPC afford it?
2. Is it reasonable to have it? (as in, does this ship show up predominantly in areas where the ramscoop is available?)
3. Is it better for gameplay if the afterburner raven now has infinite afterburner?
4. Are there alternative options that would make for better gameplay?

Number 3 is probably the biggest one. Currently, the afterburner is a finite ressource. They can use it for a while, but they'll eventually run out. This gives the player the tactical option of out-lasting them by dodging and running. If they have a ramscoop, then they'll always have more afterburner to use; and it's just the frequency/intensity of usage that will decrease after a certain time period. 

Personally, I'd probably prefer to use that space for additional fuel pods rather than a ramscoop; as that would give them a little more flexibility, but they'd still have a hard  "end" to their ability to afterburn consistently.

**Comment by LorenzoBolla** [comment]
I had already thought of everything you said, and should have  chosen to include them in the description as opposed to opting to have it be short for ease of reading, as i tend to delve into too much detail otherwise. Oops! My bad, Z.

To answer your questions:
1 - Definitely, the Ramscoop is exceedingly cheap in comparison to the ship it is on and this ships is not used by particularly struggling pilots like some others may be.

2 - Definitely, it almost only shows up where they are available. It often shows up as a Deep ship in the Deep or in service of the Deep. Even when it very rarely shows up elsewhere as a bounty hunter or pirate, it's still often within range (in the north) to easily purchase ramscoops and, either way, the ship is only sold where ramscoops are also sold. 

3 - I tested it and it's extremely unlikely, with the current afterburner AI mechanics, for it to make any functional difference whatsoever, seeing as the Volcano afterburner has low fuel burn and the Raven has a fuel capacity of 500. The AI does not afterburn enough for it to matter, at the moment, with such stats mentioned previously, in my experience testing afterburners.

The reason i have chosen to make this change is that it just makes sense, lore wise, that it would care to install the readily available where it is found ramscoop to recoup it's own fuel! But sadly, unless it's the player using it, the AI is fuel conservative enough otherwise that it never runs out of fuel. That is my opinion of afterburning AI based on my last couple of playthroughs in the last 3 months, where i paid a lot of attention to afterburners and did quite a bit of testing - my escorts barely use them, and so do enemies, when i use them far more often myself on my own flagship. Only in special occasions will they use it a decent amount more.

4 - Due to already having 500 fuel, and due to the relatively low fuel burn of the Volcano Afterburner, a single additional fuel tank (it has 14 tons leftover) would be equally unnoticeable in gameplay as the Ramscoop. I would be okay with adding it instead, if you strongly prefer it, but functionally, it is exceedingly unlikely for it to matter. It's more of a "lore" logic thing than anything else. 

In conclusion - oops, should've included this in the description rather than trying to keep it short! I tend to write too much, but this experiment at trying the opposite sure didn't work. Ah well. These are my thoughts on why it is mechanically irrelevant, but in-universe-logic correct, to add a ramscoop, and hence why i choose to do so with the empty space. If you still prefer the fuel pod, i am fine with that as well as it does also make sense. It's really just a logic thing, for me, and i had taken those 4 points into consideration as i made this, in spite of holding myself from making a small essay about it lol - sorry! Will do so in the future.

**Comment by Zitchas** [comment]
Thanks for all the detail! I very much appreciate getting all the thoughts going into it.

Given that we hope to improve the AI's combat handling (and hopefully more intelligent use of afterburners), I think I'd like to request that we go for the fuel pods instead of ramscoop here, for three reasons.

a) Fuel pods are more dependable and useful for instantaneous and high fuel draws. (ramscoops may yield almost nothing in some systems or away from the star)
b) Ramscoops are generally perceived as being exploration things only used on ships that go for longer distances away from ports. I would prefer to retain this distinction.
c) Assuming we get better AI handling of afterburners, I think it'd be better for them to be limited use resources, rather than infinite ones.

**Comment by LorenzoBolla** [comment]
Good points, particularly on the future developments, as that's something I'd be quite interested in, personally. I have changed it to a fuel pod and edited the title accordingly, then! Thanks!

**Review by Zitchas — APPROVED** [review]
Looks like an appropriate addition. Thanks!
